README.md

```diff
@@ -1721,7 +1720,7 @@ regarding stock awards and stock options.
   Technically, it cannot be made on the receipt of a stock *option* itself:
   You first must exercise that option, then file the election.
 - If you receive an early exercisable stock option (when you don’t have to wait for the the
-  stock to vest), you can make an 83(b) election on receipt of the shares on exercise.
+  stock to vest), you can make an 83(b) election at receipt of the exercised shares.
```

Comment:
I believe every lawyer would say "on" or "upon" here, not "at".

Reply:
Fixed ("upon")